packages/react-native-sdk/docusaurus/docs/reactnative/05-ui-cookbook/13-livestream.mdx

```diff
@@ -0,0 +1,195 @@
+---
+id: livestream
+title: Livestream
```

Comment:
I would keep these titles same as other SDK docs

Reply:
I would have also done that, but other SDKs don't add a cookbook or mention anything about Host View. Since we have that component and integrators should now how to customize it, I have kept a generic name. Wdyt?

Reply:
https://getstream.io/video/docs/react/ui-cookbook/watching-a-livestream/

Reply:
Fixed @vishalnarkhede